### ISSUE TITLE
Linux - v20.0 Unable to move to completed folder when containing spaces.

### ISSUE THREAD

**Issue by myAccountIsAThrowAway** [nzbget/nzbget#570]
When downloading I'm getting "Move: failure" for nzbs that have spaces in their name (ex: test space.nzb). If I upload an nzb without spaces in the name (test.nzb) it works fine. The intermediate folders with spaces are created fine; it just fails on move.

> Sat Aug  4 18:39:25 2018        DETAIL  Successfully downloaded test space/my linux distro.iso [233/340]
Sat Aug  4 18:39:25 2018        DETAIL  Writing articles for test space/my linux distro.iso
Sat Aug  4 18:39:25 2018        INFO    Successfully downloaded test space/my linux distro.iso
Sat Aug  4 18:39:25 2018        INFO    Collection test space completely downloaded
Sat Aug  4 18:39:25 2018        INFO    Queueing test space for post-processing
Sat Aug  4 18:39:25 2018        INFO    Checking renamed files for test space
Sat Aug  4 18:39:25 2018        INFO    Loaded par2-file my linux distro.iso.par2 for par-rename
Sat Aug  4 18:39:25 2018        INFO    No renamed files found for test space
Sat Aug  4 18:39:25 2018        INFO    Checking renamed archive files for test space
Sat Aug  4 18:39:25 2018        INFO    No renamed archive files found for test space
Sat Aug  4 18:39:25 2018        INFO    Nothing to unpack for test space
Sat Aug  4 18:39:26 2018        INFO    Cleaning up test space
Sat Aug  4 18:39:26 2018        INFO    Deleting file my linux distro.iso.par2
Sat Aug  4 18:39:26 2018        INFO    Cleanup for test space successful
Sat Aug  4 18:39:26 2018        INFO    Moving completed files for test space
Sat Aug  4 18:39:26 2018        ERROR   Could not create directory /opt/nzbget/downloads/complete/test space: could not create directory /opt/nzbget/downloads/complete/test space:
Sat Aug  4 18:39:26 2018        ERROR   Move for test space failed
Sat Aug  4 18:39:26 2018        INFO    Collection test space added to history

This doesn't appear to happen in the nzbget-20.0-testing-r2190-bin-linux.run release for me. I'm running   Linux Mint 19 "Tara" - Xfce (64-bit). Is this a bug in v20 or is there some setting that I've missed?

**Comment by hugbug** [comment]
There is no much difference between r2190 and stable. Is your stable installed via official installer from nzbget download page?

Does nzb really have name "test space" or you altered the log? Because if you altered it the problem can be in something else.

You can install debug version (see github releases area) and activate all debug options (DebugTarget etc.). The debug version may log more Infos.

**Comment by myAccountIsAThrowAway** [comment]
I changed the file name mentioned in the snippet above, the name of the nzb was real. I renamed an nzb to "test space" and manually uploaded it and it failed on move. Stable was pulled from: 

>  wget https://nzbget.net/download/nzbget-latest-bin-linux.run

I'll grab the debug version and see if it can produce any more info and if it fails I'll provide an unadulterated log.

**Comment by myAccountIsAThrowAway** [comment]
Well after testing debug in a local directory everything was fine. I then tested debug in my /opt/nzbget directory and all was well... so I reverted to nzbget-20.0-bin-linux.run ... it works fine. Must have been an ownership issue when I created the directories; I'm still confused because when I su to my nzbget user and moved them manually they were fine. RIP ME.

I will close this as a PEBKAC issue. Thanks for your assitance!
